src/gluonts/model/rotbaum/_estimator.py

```diff
@@ -38,11 +42,28 @@ class ThirdPartyEstimator(Estimator):
     def __init__(self, predictor_cls: type, **kwargs) -> None:
         self.predictor = predictor_cls(**kwargs)
 
+    @override()
     def train(
         self, training_data: Dataset, validation_dataset=None
     ) -> Predictor:
         return self.predictor(training_data)
 
+    def train_model(self):
+        return TrainOutput(
+            transformation=self.create_transformation(),
+            trained_net=self.create_training_network(),
+            predictor=self.create_predictor(),
+        )
+
+    def create_predictor(self) -> Predictor:
+        return self.predictor
+
+    def create_transformation(self) -> Transformation:
```

Comment:
Would we want to add any transformations here like in the other models with the time and age features?

Reply:
I was unsure if we wanted that, but when looking into it, it looked like transformations were passed into training networks which would require us to redo the architecture of Rotbaum. I'm not sure if there is another way to work with transformations, but can discuss this offline.